engine/rest-client/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>DataCleaner-engine</artifactId>
+		<groupId>org.eobjects.datacleaner</groupId>
+		<version>4.1-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>DataCleaner-rest-client</artifactId>
+
+	<dependencies>
+		<!-- REST client -->
+		<dependency>
+			<groupId>com.sun.jersey</groupId>
+			<artifactId>jersey-client</artifactId>
+			<version>1.19</version>
```

Comment:
I hope that 1.9 is compatible with 1.19. But if not then we need to downgrade here because this functionality should also be compatible with the rest I guess.